parts/footer.html

```diff
@@ -0,0 +1,97 @@
+    <!-- /wp:column -->
+    
+    <!-- wp:column {"verticalAlignment":"top","width":"45%","style":{"spacing":{"blockGap":"0","padding":{"right":"0","left":"0"}}}} -->
+    <div class="wp-block-column is-vertically-aligned-top" style="padding-right:0;padding-left:0;flex-basis:45%"><!-- wp:group {"className":"dswp-align-right-desktop-left-mobile","style":{"spacing":{"blockGap":"var:preset|spacing|80","margin":{"top":"0","bottom":"0"},"padding":{"right":"0","left":"0"}}},"layout":{"type":"flex","flexWrap":"nowrap","verticalAlignment":"top"}} -->
+    <div class="wp-block-group dswp-align-right-desktop-left-mobile" style="margin-top:0;margin-bottom:0;padding-right:0;padding-left:0"><!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+    <div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:heading {"level":6,"style":{"spacing":{"margin":{"top":"0","bottom":"var:preset|spacing|30"}}}} -->
+    <h6 class="wp-block-heading" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--30)">Organization</h6>
+    <!-- /wp:heading -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/" data-type="link" data-id="https://www2.gov.bc.ca/">TI Corp</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca">Work at TI Corp</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/gov/content/home/disclaimer">Projects</a></p>
+    <!-- /wp:paragraph --></div>
+    <!-- /wp:group -->
+    
+    <!-- wp:group {"style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+    <div class="wp-block-group" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:heading {"level":6,"style":{"spacing":{"margin":{"top":"0","bottom":"var:preset|spacing|30"}}}} -->
+    <h6 class="wp-block-heading" style="margin-top:0;margin-bottom:var(--wp--preset--spacing--30)">Contact us</h6>
+    <!-- /wp:heading -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/" data-type="link" data-id="https://www2.gov.bc.ca/"></a><a href="https://www2.gov.bc.ca/" data-type="link" data-id="https://www2.gov.bc.ca/">Phone: 250-940-8516</a></p>
+    <!-- /wp:paragraph -->
+    
+    <!-- wp:paragraph {"style":{"typography":{"lineHeight":"3"}}} -->
+    <p style="line-height:3"><a href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca"></a><a href="https://www2.gov.bc.ca/gov/content/about-gov-bc-ca">Email: Admin@ticorp.ca</a></p>
```

Comment:
why is the href pointing to a webpage, but the a tag content is an email address?

Reply:
Will updated when I the links, should have added empty links.